keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint/kep.yaml

```diff
@@ -7,10 +7,10 @@ participating-sigs:
   - sig-cluster-lifecycle
 status: implementable
 creation-date: 2020-10-03
+last-updated: 2022-01-10
 reviewers:
   - "@fabriziopandini"
 approvers:
   - "@fabriziopandini"
-
-latest-milestone: "v1.20"
+latest-milestone: "0.0"
```

Comment:
this is something that has been done in other KEPs to prevent unit tests from failing with a requirement for production readiness YAML file to exist under https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-cluster-lifecycle

we can just track release versions in `## Implementation History`